be/src/exec/exec_node.cpp

```diff
@@ -339,7 +337,7 @@ Status ExecNode::create_tree_helper(
     }
 
     if (!node->_children.empty()) {
-        node->runtime_profile()->add_child(node->_children[0]->runtime_profile(), false, NULL);
```

Comment:
It seem that the `false` here is to distinguish child[0] and other children.
Are you sure this is more readable? You can put the result screenshots here to explain.

Reply:
after change the ```true```, we can eaily know olap_scan_node is child of aggregation_node. 
![image](https://user-images.githubusercontent.com/10553413/80442620-e7048700-893f-11ea-84fa-6d590b48f99f.png)